bundle/manifests/logging.openshift.io_clusterlogforwarders_crd.yaml

```diff
@@ -13,7 +13,7 @@ spec:
     shortNames:
     - clf
     singular: clusterlogforwarder
-  scope: Cluster
+  scope: Namespaced
```

Comment:
Isn't this part of a PR already merged?

Reply:
Not sure what changes got picked up here. I didn't touch that file in this PR. Have a look at the rebase.